app/routes/users/components/Penalties.js

```diff
@@ -43,9 +43,9 @@ function Penalties({
                     {penalty.weight} {word}
                   </strong>
                 </div>
-                <div>
+                <>
```

Comment:
What is this?

Reply:
React fragments :heart_eyes: :heart: :heart_eyes_cat:

Reply:
https://reactjs.org/docs/fragments.html

Only available now with babel7

Reply:
Agreed! Did you just pop one in to test? 👍

Reply:
Hahah, yee. Nice to have when you don't want to return a div, just a set of components.